test/e2e/clusterautoscaler_helpers.go

```diff
@@ -110,8 +110,7 @@ func WaitForClusterAutoscalerToBeReadyForWorkloadCluster(
 				ObjectMeta: metav1.ObjectMeta{
 					Namespace: input.WorkloadCluster.Namespace,
 					Name: fmt.Sprintf(
-						"%s-%s",
-						clusterAutoscalerReleaseName,
```

Comment:
I think you want to keep `clusterAutoscalerReleaseName` right?

Reply:
No, the deployment is called `cluster-autoscaler-*`, only the HCP is called `ca-*`. Slightly confusing for sure...